exercises/arrays/sum/sum.js

```diff
@@ -10,17 +10,26 @@
  */
 function sum(array) {
   // This is your job. :)
-  let total = array[0]
-
-  //for (i = 0; i < array.length; i++) {
-    Math.[array]
+  let sum = 0
+  for (let i = 0; i < array.length; i++) {
+    sum = sum + Number(array[i])
```

Comment:
You can simplify this line with `sum += Number(array[i])`. Also, another commonly used built-in function to make the string->number conversion is `.parseInt()`